code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -122,6 +121,21 @@
 	new /obj/item/weapon/reagent_containers/syringe( src )
 	new /obj/item/weapon/reagent_containers/syringe( src )
 
+/obj/item/weapon/storage/box/lethalsyringes
+	name = "box of lethal injection syringes"
+	desc = "A box full of syringes."
+	icon_state = "syringe"
+
+/obj/item/weapon/storage/box/lethalsyringes/New()
+	..()
+	new /obj/item/weapon/reagent_containers/syringe/lethal(src)
+	new /obj/item/weapon/reagent_containers/syringe/lethal(src)
+	new /obj/item/weapon/reagent_containers/syringe/lethal(src)
+	new /obj/item/weapon/reagent_containers/syringe/lethal(src)
+	new /obj/item/weapon/reagent_containers/syringe/lethal(src)
+	new /obj/item/weapon/reagent_containers/syringe/lethal(src)
+	new /obj/item/weapon/reagent_containers/syringe/lethal(src)
+

```

Comment:
Unless I'm mistaken you can't actually fill lethal syringes once they're empty, so giving someone a box of empty ones is useless